biojava-core/src/test/java/org/biojava/nbio/core/sequence/io/embl/EmblParserTest.java

```diff
@@ -0,0 +1,26 @@
+package org.biojava.nbio.core.sequence.io.embl;
+
+import org.junit.Test;
+
+import java.io.File;
+/**
+ * This class should test the parser of EmblParser class
+ * @author Noor Aldeen Al Mbaidin
+ */
+public class EmblParserTest {
+
+    @Test(expected = NullPointerException.class)
+    public void givenNullFileParameterWhenCreateEmblParserThenthrowException(){
+        File file = null;
+        EmblParser emblParser = new EmblParser(file);
+    }
+
+    @Test
+    public void test(){
+        File file = new File("/home/pslpt219/Desktop/Homo.dat");
```

Comment:
This won't work for any other user. You will need to add the file to the main/resources folder and read it from there. See how it is done elsewhere in biojava. If the file is very big, please gzip it before adding it.